trajoptlib/CMakeLists.txt

```diff
@@ -233,3 +233,20 @@ if(BUILD_EXAMPLES)
         endif()
     endforeach()
 endif()
+
+if (CMAKE_VERSION VERSION_LESS 3.18)
+  set(DEV_MODULE Development)
```

Comment:
This branch isn't needed because the minimum required CMake version is already declared to be 3.22.